code/src/main/java/com/onlinecontacttracing/servlets/delete-data/DeletePositiveUserLocationsServlet.java

```diff
@@ -0,0 +1,19 @@
+package com.onlinecontacttracing.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+@WebServlet("/delete-old-positive-user-locations")
+public class DeletePositiveUserLocationsServlet extends HttpServlet {
+
+  static final Logger log = Logger.getLogger(DeletePositiveUserLocationsServlet.class.getName());
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    OldDataDeleter.deleteOldData(OldDataDeleter.POSITIVE_USER_LOCATIONS, log);
+  }
+}
```

Comment:
these all have the same code structure but I guess there's not way to abstract that out?

Reply:
I think it is abstracted out with the OldDataDeleter class. Do you have any suggestion how it could be abstracted even more?

Reply:
I guess not really, just thought I'd ask